### DISCUSSION TITLE
如何构建FDP

### DISCUSSION THREAD

**Discussion by 03kill** [SkidderMC/FDPClient#1015] [Q&A]
作者能否出个详细的FDP构建过程使用IntelliJ IDEA 2023.2.2

**Comment by Minnie123special** [comment]
<img width="736" height="41" alt="image" src="https://github.com/user-attachments/assets/61ced553-fcf9-4a0c-9d72-43ffa3e18419" />
盲人试图寻找眼珠子

**Comment by Minnie123special** [comment]
gradlew.bat build -x generateGitProperties

**Comment by Minnie123special** [comment]
https://adoptium.net/en-GB/temurin/releases?version=8&os=windows&arch=x64&mode=filter
